config-solaris.md

```diff
@@ -0,0 +1,88 @@
+# Solaris Application Container Configuration
+
+Solaris application containers can be configured using the following properties, all of the below properties are the same as given in zonecfg(1M) man page, except milestone. The Solaris specification is entirely optional.
+
+## milestone
+The SMF(Service Management Facility) FMRI which should go to "online" state before we start the desired process within the container.
+
+**`milestone`** *(string, optional)*
+
+### Example
+```json
+"milestone": "svc:/milestone/container:default"
+```
+
+## limitpriv
+The maximum set of privileges any process in this container can obtain. The property should consist of a comma-separated privilege set specification as described in priv_str_to_set(3C) man page for the respective release of Solaris.
+
+**`limitpriv`** *(string, optional)*
+
+### Example
+```json
+"limitpriv": "default"
+```
+
+## max-shm-memory
+The maximum amount of shared memory allowed for this application container. A scale (K, M, G, T) can be applied to the value for each of these numbers (for example, 1M is one megabyte).
+
+**`max-shm-memory`** *(string, optional)*
+
+### Example
+```json
+"max-shm-memory": "512m"
+```
+
+## capped-cpu
+Sets a limit on the amount of CPU time that can be used by a container. The unit used translates to the percentage of a single CPU that can be used by all user threads in a container, expressed as a fraction (for example, .75) or a mixed number (whole number and fraction, for example, 1.25). An ncpu value of 1 means 100% of a CPU, a value of 1.25 means 125%, .75 mean 75%, and so forth. When projects within a capped container have their own caps, the minimum value takes precedence.
+
+* **`ncpus`** *(string, optional)*
```

Comment:
On Fri, Apr 29, 2016 at 01:08:38PM -0700, Abhijeeth Nuthan wrote:
> > +* **`ncpus`** *(string, optional)*
>
> @wking : I'm assuming you are suggesting adding an (object,
> optional) for capped-cpu and capped-memory, etc. That would require
> rework for all the properties even in config-linux.md . I prefer not
> to do that in this PR.

I feel like we can work towards that goal in baby steps instead of
shifting everything at once.  But yeah, if it doesn't feel right to
you, we can certainly defer until someone is bothered enough to PR the
change.